Mol_Analysis.py

```diff
@@ -434,7 +457,8 @@ def runall(self, **kwargs):
             self.getinfo(self.outf)
 
         if self.torun["mindist"] is True:
-            self.mindist()
+            num_cores = kwargs.get("nc", args.nc)
```

Comment:
this is a bit strange to me - here just using a function argument would be enough. If you _need_ to do this for it to work, it suggests that there's an issue in flow control for this variable, i.e. how it passes through the different functions during execution